config.cc

```diff
@@ -2204,7 +2211,7 @@ QString getConfigDir() THROW_SPEC( exError )
 
 QString getIndexDir() THROW_SPEC( exError )
 {
-  QDir result = getHomeDir();
+  QDir result = getCacheDir();
```

Comment:
This is an unrelated change. It forces rebuilding the entire index. Plus disk space is wasted in the old index location. Please revert. If you really want this change, ensure backwards compatibility as in `getHomeDir()` in a future pull request.